model/src/main/java/io/syndesis/model/connection/Action.java

```diff
@@ -30,13 +30,16 @@
 import io.syndesis.model.WithId;
 import io.syndesis.model.WithName;
 import io.syndesis.model.WithTags;
+
 import org.immutables.value.Value;
 
 @Value.Immutable
 @JsonDeserialize(builder = Action.Builder.class)
 @JsonIgnoreProperties(value = {"properties", "inputDataShape", "outputDataShape"}, allowGetters = true)
 public interface Action extends WithId<Action>, WithName, WithTags, WithConfigurationProperties, Serializable {
 
+    enum EI_Pattern { from, to }
```

Comment:
Why not just `Pattern` `EI_Pattern` is so un-Java like? Perhaps `Use`?

In Camel these values are capitalized I wonder if it would be easier to have them like that also. It would save us some translation when needed. (e.g. `From` and `To`).